source/js/main.js

```diff
@@ -109,12 +109,12 @@
     }
     autoHideMenus();
 
-    $(window).resize(function () {
+    $(window).on('resize', function () {
         autoHideMenus();
     });
 
     // Fold second-level menu
-    $('.main-nav-list-item').hover(function () {
+    $('.main-nav-list-item').on('hover', function () {
```

Comment:
This breaks the main navbar dropdown.

Reply:
What exactly is broken? How do I reproduce?
https://marcofranssen.nl/ seems to be working fine.

Can you attach screenshot of the issue?

Reply:
![image](https://user-images.githubusercontent.com/8849362/42851092-6c1ed0ae-89f0-11e8-9ea7-929ed30de06c.png)
Dropdown menu no longer expands when hovered. I already fixed it.

Reply:
aaah. I don't have dropdowns on my side :( thanks for fixing. Sorry I missed it out.